src/scripts/pages/home.js

```diff
@@ -47,12 +50,13 @@ document.addEventListener('DOMContentLoaded', () => {
   function toggleCategories () {
     const storageValue = sessionStorage.getItem(storageKey);
 
-    if (storageValue === showingPrivate) {
-      sessionStorage.setItem(storageKey, showingPublic);
-    } else if (storageValue === showingPublic) {
-      sessionStorage.setItem(storageKey, showingPrivate);
+    if (storageValue) {
+      if (storageValue === showingPrivate) {
+        sessionStorage.setItem(storageKey, showingPublic);
+      } else if (storageValue === showingPublic) {
+        sessionStorage.setItem(storageKey, showingPrivate);
+      }
     }
```

Comment:
Is the `if (storageValue) {}` needed? I can't see how it would prevent an error, since we aren't accessing any properties on `storageValue`.

If `storageValue` was null, both conditions would evaluate false and `sessionStorage.setItem(...` wouldn't be called. I could be missing something though.

```suggestion
  if (storageValue === showingPrivate) {
    sessionStorage.setItem(storageKey, showingPublic);
  } else if (storageValue === showingPublic) {
    sessionStorage.setItem(storageKey, showingPrivate);
  }
```

Reply:
I think this was just a mindless change on my part. Removing the extra `if (storageValue) {}` makes sense to me.